sources/packages/web/src/components/generic/CheckAESTPermissionRole.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <slot :isReadonly="isReadonly"></slot>
+</template>
+
+<script lang="ts">
+import { computed, PropType } from "vue";
+import { ClientIdType, Role } from "@/types";
+import { AuthService } from "@/services/AuthService";
+
+export default {
+  props: {
+    role: {
+      type: String as PropType<Role>,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const isReadonly = computed(() => {
+      const userToken = AuthService.shared.userToken;
+      if (userToken?.resource_access && userToken?.azp) {
```

Comment:
Can we move this logic to the composable `useAuth`?
It can return just the array of roles for the current `azp`.

Reply:
It can also be a simple method `hasHole(<some_role>)` i the composable.